lib/scenic/scene.ex

```diff
@@ -195,10 +195,10 @@ defmodule Scenic.Scene do
   to the cursor input, the text field captures text input (and
   maybe transforms its graph to show that it is selected).
 
-  Captured input types are should be released when no longer
+  Captured input types should be released when no longer
```

Comment:
So on capturing input with the `handle_input/3` callback
does this mean it is blocking until
`{:noreply, state :: any()}` is returned?
Or is this releasing referring to something else?

Reply:
I'll give it a good read in a bit. Either way it clearly needs attention!